lib/specinfra/helper/detect_os/clearlinux.rb

```diff
@@ -0,0 +1,12 @@
+class Specinfra::Helper::DetectOs::Clearlinux < Specinfra::Helper::DetectOs
+  def detect
+    if run_command('swupd info').success?
+      release = nil
+      swupd_info = run_command("swupd info")
+      swupd_info.stdout.each_line do |line|
```

Comment:
This would be better because of fewer `run_command()`.

```ruby
  def detect
    swupd_info = run_command('swupd info')
    if swupd_info.success?
      release = nil
      swupd_info.stdout.each_line do |line|
```

Reply:
Good point, thanks, fixed.

Reply:
Thanks!